### ISSUE TITLE
Cannot read property 'versions' of undefined

### ISSUE THREAD

**Issue by HitomiTenshi** [tjunnone/npm-check-updates#608]
- [x] I have read the list of [known issues](https://github.com/tjunnone/npm-check-updates#known-issues) before filing this issue.
- [x] I have searched for [similiar issues](https://github.com/tjunnone/npm-check-updates/issues) before filing this issue.

* node version: 13.0.1
* npm version: 6.13.0
* npm-check-updates version: 3.2.0

---------------------------

Running `ncu -n` results in the following error when scanning one of my projects:

```
C:\Users\fegit1\AppData\Roaming\npm\node_modules\npm-check-updates\lib\npm-check-updates.js:392
    throw err;
    ^

TypeError: Cannot read property 'versions' of undefined
    at C:\Users\fegit1\AppData\Roaming\npm\node_modules\npm-check-updates\lib\package-managers\npm.js:226:64
    at async Promise.all (index 22)
    at async getAnalysis (C:\Users\fegit1\AppData\Roaming\npm\node_modules\npm-check-updates\lib\npm-check-updates.js:438:30)
    at async Object.run (C:\Users\fegit1\AppData\Roaming\npm\node_modules\npm-check-updates\lib\npm-check-updates.js:444:12)
```

Running `ncu` without the `-n` argument runs fine.

Doesn't happen on all of my projects, only some.

The previous version `v3.1.26` does not have this issue.

**Comment by raineorshine** [comment]
Thank you for reporting! Can you please narrow it down to which dependency(ies) is causing the issue? Once you find it, please post the dependency name and version specified in your package.json so I can reproduce.

**Comment by HitomiTenshi** [comment]
@raineorshine I just found the issue. It's a git repo dependency.

Run:  
`npm init` (yes to all)  
`npm install github:HitomiTenshi/pouchdb-browser --save`  
`ncu -n`

**Comment by raineorshine** [comment]
@webdevium It looks like there is a regression from your PR. Could you look into this?

**Comment by webdevium** [comment]
@raineorshine My PR, my regression, my fix

**Comment by raineorshine** [comment]
Thanks!

Published in `v3.2.1`

@HitomiTenshi Did this fix the issue?

**Comment by HitomiTenshi** [comment]
@raineorshine Yes it did, thanks!
